### ISSUE TITLE
Search tool - selecting a layer

### ISSUE THREAD

**Issue by SeritavdW** [iobis/bioeco-geonode#132]
When searching for a network/ keyword, the available options pop up, but clicking on an option nothing happens, only after the second click, it goes to the page. With the first click the dropdown menu only becomes narrower but does not go to the page.

**Comment by pieterprovoost** [comment]
@SeritavdW I can't reproduce the problem, is this about the text search at https://geonode.goosocean.org/search?

**Comment by SeritavdW** [comment]
Yes, for some, the first click takes you to the layer, for others only after the second click. Test searching "lifewatch" only, then select "Zooplankton observations in the Belgian part of the North Sea". Does it not happen when you try?

**Comment by pieterprovoost** [comment]
No issues here, I'll check on your machine.
